SPECS/containerized-data-importer/containerized-data-importer.spec

```diff
@@ -155,7 +168,7 @@ install -m 0644 _out/manifests/release/cdi-cr.yaml %{buildroot}%{_datadir}/cdi/m
 %files api
 %license LICENSE
 %doc README.md
-%{_bindir}/virt-cdi-apiserver
+%{_bindir}/cdi-apiserver
```

Comment:
This way we're effectively removing the binaries we have already made public and potentially introducing regressions fo whoever has already been using this package. Are we only re-naming the files but they are actually the same executables or are the new ones different from the old ones?

Reply:
Agreed, would like to see symlinks from the old names to the new.

Reply:
Same binaries, just renames. 
We had a bad name to begin with. Those were created mainly for [cdi containers](https://github.com/kubevirt/containerized-data-importer/) so less likely a chance of breaking anyone.

Reply:
Yes, but only if the new files are still the same executables, just with a different name. If these are different tools, then we'd need to keep both.